### ISSUE TITLE
Emissive texture not exported

### ISSUE THREAD

**Issue by Ali-RS** [KhronosGroup/glTF-Blender-IO#57]
Hi 
When exporting my model with this add-on, Emissive texture is not exported. 

Here is my node setup 

![screenshot_2018-10-28_11-48-16](https://user-images.githubusercontent.com/5787647/47613586-73ae5a80-daa7-11e8-9d1c-0d4305d3372a.png)

inside exported gltf file I can see the diffuse and metallic roughness textures are listed but no emissive texture. 

```
   "images" : [
        {
            "name" : "diffuse",
            "uri" : "diffuse.png"
        },
        {
            "name" : "metallic-roughness",
            "uri" : "metallic-roughness.png"
        }
    ],
```
And after loading it in my game engine editor , I can see diffuse and metallic roughness map is there but emissive map slot is empty.  

Am I doing anything wrong? 
Will appreciate any help.

**Comment by UX3D-nopper** [comment]
Let us check, we will have a look at it.

**Comment by UX3D-becher** [comment]
Should be fixed in  21d97dc

**Comment by Ali-RS** [comment]
Thanks
